### PR TITLE
New updater options for v8.6.9

### LINKED ISSUE #678: installer '/closeRunningNpp' cmdline option
Following the PR: https://github.com/notepad-plus-plus/notepad-plus-plus/pull/15230

For the possible future description please look here: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8514#issuecomment-2156756428

### PR DESCRIPTION
**Pull request by pryrt**
closes #678

(Waiting for v8.6.9-RC Announcement before merging PR)

### CODE REVIEW

File: content/docs/command-prompt.md [hunk 1]
```diff
@@ -200,6 +200,12 @@ The Notepad++ [installer executable](../getting-started/#installer) accepts the
 - `/D=c:\blah` or `/D=c:\path with spaces\blah` : Overrides the default installation directory.
     - Do _not_ put quotes around the path, even when there are spaces.
     - Because it allows spaces in the path, this option **must** be the last argument on the installer command line, if included.
+- `/closeRunningNpp`: Will (try to) close existing Notepad++ before installing the new version.  (New to v8.6.9.)
+    - It will first try a "nice" close request (using [WM_CLOSE](https://learn.microsoft.com/en-us/windows/win32/winmsg/wm-close)).
+    - If the "nice" request wasn't successful after 5 seconds, it will kill the underlying process.
+    - If Notepad++ has multiple instances opened, it will close all instanaces.
```
Comment (xomx) [inline content/docs/command-prompt.md:206]: instanaces -> instances

File: content/docs/upgrading.md [hunk 2]
```diff
@@ -12,6 +12,22 @@ If you used installation package when you originally installed, the Auto Update
...
 
+### Updater Dialog
+
+The Updater dialog is launched for either Auto-Trigger or for Upgrade On-Demand.
+
+If there is no auto-triggered update available, the dialog will just inform you of this (but see [No New Version Found](#no-new-version-found-safety-delay)).
+
+If there is a new version, the Updater dialog will allow you to choose:
+- **Yes**: Run the update.
+    - It will download the updater, ask if it's okay to close Notepad++, and the updater will run, asking you to confirm your setttings, and update Notepad++.
```
Comment (xomx) [inline content/docs/upgrading.md:23]: setttings -> settings

File: content/docs/command-prompt.md (hunk 1 above)
Comment (pryrt) [reply content/docs/command-prompt.md:206]: thanks.  fixed.

File: content/docs/upgrading.md (hunk 2 above)
Comment (pryrt) [reply content/docs/upgrading.md:23]: Thanks.

File: content/docs/command-prompt.md [hunk 3]
```diff
@@ -200,6 +200,12 @@ The Notepad++ [installer executable](../getting-started/#installer) accepts the
 - `/D=c:\blah` or `/D=c:\path with spaces\blah` : Overrides the default installation directory.
     - Do _not_ put quotes around the path, even when there are spaces.
     - Because it allows spaces in the path, this option **must** be the last argument on the installer command line, if included.
+- `/closeRunningNpp`: Will (try to) close existing Notepad++ before installing the new version.  (New to v8.6.9.)
+    - It will first try a "nice" close request (using [WM_CLOSE](https://learn.microsoft.com/en-us/windows/win32/winmsg/wm-close)).
+    - If the "nice" request wasn't successful after 5 seconds, it will kill the underlying process.
+    - If Notepad++ has multiple instances opened, it will close all instances.
+- `/runNppAfterSilentInstall`: After a silent install, it will automatically run the newly-installed Notepad++. (New to v8.6.9.)
+    - Only works if `/S` is also specified.
```
Comment (xomx) [inline content/docs/command-prompt.md:208]: @pryrt 

I think that these two new options belongs to the next "It also implements an additional option:"  (and it should be updated to `It also implements additional options:`) part and not to the "The Notepad++ installer executable accepts the `three` NSIS command-line options:"

And the [installer executable](https://github.com/notepad-plus-plus/npp-usermanual/blob/18d2bd6808c1a69c3e8c09440635e5a724adf7a5/content/getting-started/#installer) link there leads to `404 - page not found`

**Pull request merged**